### PR TITLE
[master] Update scope assignment docs for admin portal

### LINKED ISSUE #2517: Confusions in Role Permissions in Admin Portal
**Description:**
A customer has confused the role permissions in management console with the role-scope assignments in the admin portal. The docs related to this should be updated as mentioned in https://github.com/wso2/product-apim/issues/9572

**Suggested Labels:**
CEXP

**Affected Product Version:**
3.2.0

### PR DESCRIPTION
**Pull request by RakhithaRR**
## Purpose
This PR updates the scope assignments (previously role permissions) related docs in the admin portal.

Fixes #2517
Related issue: https://github.com/wso2/product-apim/issues/9572

### CODE REVIEW

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 1]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
...
         
-        ![Add Role Mapping]({{base_path}}/assets/img/administer/new-role-mapping.png)
+        ![Add Role Mapping]({{base_path}}/assets/img/administer/new-role-alias-mapping.png)
 
-    - **Custom Permissions** - New roles can be associated with scopes individually. If you create a custom role that has different permissions, add that role under the required scopes based on the functionality or permissions you need to give to a user carrying this role. For example, if you need to allow the user to create APIs, select the new scope `apim:api_create` under **custom permissions**.
+    - **Custom Scope Assignments** - New roles can be associated with scopes individually. If you create a custom role that has different permissions, add the required scopes for the role based on the functionality or permissions you need to give to a user carrying this role. For example, if you need to allow the user to create APIs, select the new scope `apim:api_create` under **custom scope assignments**.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:182]: we need to reword this to say that we add roles to a scope, not scopes to a role.

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 2]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
...
         
-        ![Add Scope Mapping]({{base_path}}/assets/img/administer/new-scope-mapping.png) 
+        ![Add Scope Mapping]({{base_path}}/assets/img/administer/new-custom-scope-mapping.png) 
 
 !!! info 
     The following are the scopes allowed for each default Internal role under the default configurations.
```
Comment (shaniR) [inline en/docs/administer/managing-users-and-roles/managing-permissions.md:187]: here also shouldn't we say that the roles allowed for the scope instead of the other way round?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 3]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
 </tbody>
 </table>
 
-## Adding role-based permissions
+## Adding API-M specific Scope Assignments
```
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:160]: These scopes are defined for APIM use unlike the ones that are available in the management console for registry purposes

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 4]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
...
 
-1. Sign in to Admin Portal (`https://<APIM_Host>:<APIM_Port>/admin`) and navigate to **Settings** > **Role Permissions** in the Admin Portal. 
+!!! note
+    Note that the role-scope assignments mentioned here are related to the internal API-M specific scope assignments
```
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:163]: These are defined in https://apim.docs.wso2.com/en/latest/administer/managing-users-and-roles/managing-user-roles/#adding-role-mappings

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 5]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
...
-2. Click **Add role permission**.
+2. Click **Add scope mappings**.
 
-    ![Add Role Permission]({{base_path}}/assets/img/administer/add-role-permission.png) 
+    ![Add Scope Mapping]({{base_path}}/assets/img/administer/add-scope-mapping.png) 
 
 3. Provide a role name created in a Carbon console. Then click **Next** to proceed.
 
-    ![Add Role Mapping]({{base_path}}/assets/img/administer/add-new-role-mapping.png)
+    ![Add Role Mapping]({{base_path}}/assets/img/administer/add-new-scope-name.png)
```
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:174]: yes, its the dialog that appear when we want to create a new role-scope mapping

File: en/docs/administer/managing-users-and-roles/managing-permissions.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:182]: No, we assign the scopes specific to APIM to the roles we create in the management console

File: en/docs/administer/managing-users-and-roles/managing-permissions.md (hunk 2 above)
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:187]: No, we assign the scopes specific to APIM to the roles we create in the management console

**Comment by RakhithaRR** [comment]
@shaniR please refer https://github.com/wso2/product-apim/issues/9572#issue-752026403 for more context on this

File: en/docs/administer/managing-users-and-roles/managing-permissions.md [hunk 6]
```diff
@@ -157,27 +157,31 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
...
  
-    - **Role alias** - New roles can be mapped to existing Internal/* roles, created roles, and admin. All the scopes associated with the specified role will be mapped to the new role automatically.  If you want to map the permissions of Internal/creator to the new `creator` role, select Internal/creator from the drop-down menu and save. This will update all scope mappings in the `tenant-conf.json` file with `Internal/creator` as an allowed role resulting in the new creator role to be allowed for all scopes allowed for the `Internal/creator` role.
+    - **Role alias** - New roles can be mapped to existing `Internal/*` roles, created roles, and admin. All the scopes associated with the specified role will be mapped to the new role automatically.  If you want to map the scopes of `Internal/creator` to the new `creator` role, select `Internal/creator` from the drop-down menu and save. This will update all scope mappings in the `tenant-conf.json` file with `Internal/creator` as an allowed role resulting in the new `creator` role to be allowed for all scopes allowed for the `Internal/creator` role.
```
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md:178]: Whats mentioned here is that you can map the scopes of existing roles directly to the new role you created. 
We have only one step here. i.e. selecting a role from a dropdown menu.

[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md]: Fixed with 646fcb2

File: en/docs/administer/managing-users-and-roles/managing-permissions.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RakhithaRR) [reply en/docs/administer/managing-users-and-roles/managing-permissions.md]: Fixed with 646fcb2

**Review by Mariangela — APPROVED** [review]
